pkg/apis/dashboard/v1alpha1/dashboard_spec.go

```diff
@@ -0,0 +1,14 @@
+package v1alpha1
+
+import common "github.com/grafana/grafana/pkg/apimachinery/apis/common/v0alpha1"
+
+// +k8s:openapi-gen=true
+type DashboardSpec struct {
+	Title               string `json:"title"`
+	common.Unstructured `json:",inline"`
```

Comment:
@radiohead i think there's an unmarshaling issue with `common.Unstructured` and/or ``json:",inline"`` that presented itself as a conversion issue. i "fixed" the conversion issue in #100084, but i think the real issue is that the `spec.Title` is not properly unmarshaled from storage when the object is stored as `v1alpha1`.

Reply:
Thank you! Yeah, I've noticed this when working on https://github.com/grafana/terraform-provider-grafana/pull/2006 and it's been in my backlog for now.